src/renderer/src/hooks/mutations/projects.ts

```diff
@@ -9,13 +13,22 @@ export function useCreateProject() {
 
 	return useMutation({
 		mutationKey: [CREATE_PROJECT_KEY],
-		mutationFn: (name?: string) => {
-			return api.createProject({ name })
+		mutationFn: (opts?: Parameters<typeof api.createProject>[0]) => {
+			if (opts) {
+				return api.createProject(opts)
+			} else {
+				// Have to avoid passing `undefined` explicitly
+				// See https://github.com/digidem/comapeo-mobile/issues/392
+				return api.createProject()
+			}
 		},
-		onSuccess: () => {
+		onSuccess: (projectId: string) => {
 			queryClient.invalidateQueries({
 				queryKey: getProjectsQueryKey(),
 			})
+			queryClient.invalidateQueries({
+				queryKey: getProjectSettingsQueryKey({ projectId: projectId }),
+			})
```

Comment:
this should not be needed because the invalidation call above this (using `getProjectsQueryKey()`) will cascade queries for a specific project as well

```suggestion
			queryClient.invalidateQueries({
				queryKey: getProjectSettingsQueryKey({ projectId: projectId }),
			})
```

Reply:
Ok done.